apps/www/_blog/2022-08-15-launch-week-five-cli-v1-and-admin-api-beta.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: 'Launch Week 5: CLI v1 and Admin API beta'
+description: We are moving Supabase CLI v1 out of beta, and releasing Admin API beta.
+author: soedirgo, qiao
```

Comment:
@sweatybridge you'll need to add an entry to https://github.com/supabase/supabase/blob/master/apps/www/lib/authors.json and it also looks like this only works as comma separated with spaces:
```suggestion
author: soedirgo,qiao
```